Resources/Locale/en-US/items/components/item-component.ftl

```diff
@@ -7,3 +7,9 @@ pick-up-verb-get-data-text = Pick Up
 pick-up-verb-get-data-text-inventory = Put in hand
 
 item-component-on-examine-size = Size: {$size}
+
```

Comment:
This examine text should also give the item's weight now that it also matters. I know weight is currently just based on size, but I guess that might change in the future and I don't think there's anything else to communicate that relationship with players.

Reply:
Backpack should show what size it fits for sure which is also much easier to communicate than a whitelist.